### PR TITLE
additional Trigger converter changes to enable converting high level data structures into Speedy values

### PR DESCRIPTION
**Pull request by carlpulley-da**

### CODE REVIEW

File: triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Converter.scala [hunk 1]
```diff
@@ -287,7 +296,7 @@ final class Converter(
       SUnit,
     )
 
-  def fromACS(createdEvents: Seq[CreatedEvent]): Either[String, SValue] =
+  def fromActiveContracts(createdEvents: Seq[CreatedEvent]): Either[String, SValue] =
```
Comment (carlpulley-da) [inline triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Converter.scala:299]: Renamed as the created `Value` is actually an instance of the Daml type `ActiveContracts`

File: triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Converter.scala [hunk 2]
```diff
@@ -17,24 +17,26 @@ import com.daml.lf.speedy.{ArrayList, SValue}
 import com.daml.lf.speedy.SValue._
 import com.daml.lf.value.Value.ContractId
 import com.daml.ledger.api.v1.commands.{
-  Command,
   CreateAndExerciseCommand,
   CreateCommand,
   ExerciseByKeyCommand,
   ExerciseCommand,
+  Command => ApiCommand,
```
Comment (carlpulley-da) [inline triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Converter.scala:24]: As we will be referring to both Speedy and Ledger `Command` types, aliasing the type name

File: triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Converter.scala [hunk 3]
```diff
@@ -322,6 +331,109 @@ final class Converter(
       "config" -> config,
     )
 
+  def fromCommand(command: Command): SValue = command match {
```
Comment (carlpulley-da) [inline triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Converter.scala:334]: As call sites will simply throw converter exceptions, I've intentionally not attempted to work with Either's here

File: triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Converter.scala [hunk 4]
```diff
@@ -322,6 +331,109 @@ final class Converter(
...
+      )
+
+    case _ =>
+      throw new ConverterException(
+        s"${command.getClass.getSimpleName} is an unexpected command type"
+      )
+  }
+
+  def fromCommands(commands: Seq[Command]): SValue = {
+    SList(commands.map(fromCommand).to(FrontStack))
```
Comment (carlpulley-da) [inline triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Converter.scala:380]: If `fromCommand` returned an either, we'd have to work a lot harder here (to also return an either), or convert via `orConverterException`

**Review by akrmn — APPROVED** [review]
LGTM, thank you @carlpulley-da!

**Review by remyhaemmerle-da — APPROVED** [review]

**Pull request merged**